en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,250 @@
-    
-2. On the **Services** page, click **Create**. You are directed to the **Create Service** page.
+## Step 1: Create the service resources
 
-3. Under **Create with Choreo**, enter `covid-stats` as the name of your service, and then click **Create**. You are directed to the low-code view to design the service.
+To create the service resources:
 
-## Step 2: Design the service
+1. Access the Choreo Console via [https://console.choreo.dev/](https://console.choreo.dev/).
+    
+2. Sign in using either your Google or GitHub credentials.
+    
+3. On the **Services** page, click **Create**. You are directed to the low-code view.
 
-To design the service to fetch the current active COVID-19 cases in the United States, follow this procedure:
+4. Under **Create with Choreo**, enter `CovidStatus` as the name of your service.
 
-1. Select **GET** as the HTTP method and enter `activecases` as the relative path from the host.
+5. Select **GET** as the HTTP method, and enter `stats/[string country]/[string date]` in the **Path** field.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png){.cInlineImage-half}
+    ![Resource configuration](../assets/img/services/configure-api-trigger.png){.cInlineImage-half}
+    
+    Then click **Save API**. Now you have saved the API resource that starts your service.
     
-2. Click **Save API**.  
+## Step 2: Get COVID-19 data
+
+To get COVID-19 data:
+
+1. Click on the **API Calls** tab and then click on **Covid19 API** in the available connection list.
+
+2. In the **Connection Name** field, enter `covid19Client`, and save.
+
+3. Click the last **+** icon in the low-code diagram and click **API Calls**.
 
-3. Click **API Calls** and complete the following steps to create a new HTTP connection:
+4. Under **Choose existing connection**, click **covid19Client**.
 
-    1. Click **HTTP** and enter the details as follows:
+5. In the **Operation** field, select **Country Status**. Then enter the following information in the other fields that appear.
+
+    | **Field**                 | **Value**         |
+    |---------------------------|-------------------|
+    | **Country**               | `country`         |
+    | **Response VariableName** | `statusByCountry` |
     
-        | **Field**           | **Value**                      |
-        |---------------------|--------------------------------|
-        | **Connection Name** | `httpEndpoint`                 |
-        | **URL**             | `https://api.covid19api.com`   |
+    Save the information.
     
-        ![Select HTTP Connection](../assets/img/services/select-http-connection.png){.cInlineImage-half}
-        
-    2. Click **Save**.
-        
-    3. Click the **+** icon below the HTTP API call you added. 
+6. Now let’s extract the current total cases count from the response and store it in a variable.
+
+    To do this, click the last **+** icon in the low-code diagram and click **Variable**.
```

Comment:
"In the **Statements** tab" part is missing

Reply:
Did not mention it because the **Statements** tab is open by defaults and this statement type is visible as soon as the dialog box opens.